### ISSUE TITLE
increase the percentage of test coverage

### ISSUE THREAD

**Issue by emp7yhead** [Hexlet/hexlet-friends#242]
At the moment, test coverage of the project is 64%, needed to increase the percentage of coverage.

**Comment by ajib6ept** [comment]
Беру в работу

**Comment by ajib6ept** [comment]
Да, в работе, понятно как покрытие можно улучшить, в течение недели пришлю pr

**Comment by ssssank** [comment]
@emp7yhead что думаешь? закроем этот ишью или ещё улучшим?

**Comment by EmilMorua** [comment]
@emp7yhead Добрый день! Ишью выполнено или ее можно брать?

**Comment by emp7yhead** [comment]
@EmilMorua да можно еще увеличить покрытие, так что можно брать

**Comment by EmilMorua** [comment]
Беру в работу!

**Comment by fey** [comment]
@EmilMorua Как ваши успехи?

**Comment by EmilMorua** [comment]
@fey проектом не занимаюсь пока, задача свободна

**Comment by BoCXoD-man** [comment]
Попробую взять в работу и увеличить покрытие тестами.

**Comment by ikhanter** [comment]
Попробую расширить покрытие тестами
